core/services/ardupilot_manager/mavlink_proxy/test_all.py

```diff
@@ -74,5 +74,5 @@ def test_mavlink_router() -> None:
         endpoint_2,
     ], "Endpoint list does not match."
 
-    assert mavlink_router.start(Endpoint("udp:0.0.0.0:14550")), "Failed to start MAVLinkRouter"
+    assert mavlink_router.start(Endpoint("udpout:0.0.0.0:14550")), "Failed to start MAVLinkRouter"
```

Comment:
this means that we do not support "udp" anymore ? This is a pretty standard thing between mavlink-router and mavproxy

Reply:
It means, yes.
An honest workaround would be to have a parser from `udp` to `udpout` on endpoint creation.

Reply:
I'm a bit inclined to only support `udpout` over `udp` and 'udpout`, having a single name for a single functionality makes it more coerent, but I'm not sure since mostly mavlink routers support boths names, @jaxxzer what you think about it ?

Reply:
I'm also a fan of having a single way to do it.

I always have to check the docs to see if `udp` means client or server.

Reply:
only `udpout` and `udpin` is the best for me.